### PR TITLE
Clarify expiry times for void, capture and refund

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

**Comment by ct-clearhaus** [comment]
RFR @mt-clearhaus @soltveit

File: source/index.md [hunk 1]
```diff
@@ -511,6 +511,9 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
   </dd>
 </dl>
 
+<p class="alert alert-info">
+<b>Notice:</b> A capture cannot be made after 180 days after the authorization.
```
Comment (mt-clearhaus) [inline source/index.md]: My eyes stumple upon "after" and "after". Maybe "A capture can only be made until 180 days after the authorization."?

Comment (mt-clearhaus) [inline source/index.md]: Ok. The "after" and "after" are valid, but my suggestion is also valid :-) It's up to you. I suggest similar alterations to refund and void.

**Comment by ct-clearhaus** [comment]
@soltveit will you review and deploy?

**Comment by mt-clearhaus** [comment]
:+1:

**Comment by soltveit** [comment]
👍

**Comment by soltveit** [comment]
Deployed.

**Pull request merged**